api.wordpress.org/public_html/patterns/1.0/tests/test-index.php

```diff
@@ -297,7 +292,7 @@ public function data_search_patterns() {
 	 * @group e2e
 	 */
 	public function test_search_title_match_boosted_above_description_match() : void {
-		$search_term = 'image';
+		$search_term = 'heading';
```

Comment:
"image" no longer returns patterns without the term in the title, so the array is arbitrarily resorted (all sort weights are 0). Switching to "heading" returns some items without the term in the title, so the sorting works.